includes/ApplHandlingByStr.php

```diff
@@ -20,6 +20,7 @@ function showStatus( $stat_str )
 		if( $str[1] == 'P' )
 		{
 			echo "<td><b>PENDING</b> by $usr</td>";
+			break;
```

Comment:
__[code_sniffer]__
* Spaces must be used to indent lines; tabs are not allowed

<!-- Comment by [SideCI](https://sideci.com) -->